ini/ini.go

```diff
@@ -0,0 +1 @@
+package ini
```

Comment:
Looks like this is an empty package, let's delete it

Reply:
Whoops, nice spot!